test/fixtures/v8-coverage/subprocess.js

```diff
@@ -6,3 +6,5 @@ setTimeout(() => {
     const c = 102;
   }
 }, 10);
+
+process.on('warning', console.log);
```

Comment:
Fwiw, the warning should be printed to stderr by default anyway, and using `console.log()` prints to stdout rather than stderr, so the test doesn’t use this line, right?

Reply:
Oh! I was not aware of this. This whole line is not required as warning is already getting into `stderr`. 👍